FileHeader.py

```diff
@@ -27,6 +27,8 @@
 HEADER_PATH = os.path.join(PLUGIN_PATH, 'template/header')
 BODY_PATH = os.path.join(PLUGIN_PATH, 'template/body')
 INSTALLED_PLGIN_PATH = os.path.abspath(os.path.dirname(__file__))
+DEFAULT_CUSTOM_TEMPLATE_HEADER_PATH = os.path.join(PACKAGES_PATH, 'User/FileHeader-template/header')
+DEFAULT_CUSTOM_TEMPLATE_BODY_PATH = os.path.join(PACKAGES_PATH, 'User/FileHeader-template/body')
```

Comment:
Change it to below maybe better:

`DEFAULT_CUSTOM_TEMPLATE_PATH = os.path.join(PACKAGES_PATH, 'User/FileHeader-template')
DEFAULT_CUSTOM_TEMPLATE_HEADER_PATH = os.path.join(DEFAULT_CUSTOM_TEMPLATE_PATH, 'header')
DEFAULT_CUSTOM_TEMPLATE_BODY_PATH = os.path.join(DEFAULT_CUSTOM_TEMPLATE_PATH, 'body')
`